.github/workflows/generate_manpage.yml

```diff
@@ -19,7 +19,35 @@ jobs:
           python-version: 3.x
 
       - name: Install dependencies
-        run: pip install argparse-manpage
+        run: |
+          pip install argparse-manpage
+          pip install six
+          pip install pexpect
+          pip install rpm
+
+      - name: Install python3-rpm package with apt-get
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y python3-rpm
+
+      - name: Reinstall python3-rpm
+        run: sudo apt-get install --reinstall -y python3-rpm
+
+      - name: Check RPM Module Location
+        run: python3 -c "import rpm; print(rpm.__file__)"
+
+      - name: Check RPM Module Availability
+        run: python3 -c "import rpm; print(rpm)"
+
+      - name: Verify python3-rpm installation
+        run: |
+          if python3 -c "import rpm" &>/dev/null; then
+            echo "python3-rpm is installed successfully."
+          else
+            echo "ERROR: python3-rpm installation failed."
+            dpkg -l | grep python3-rpm
+            exit 1
+          fi
```

Comment:
Thank you for the suggestion here, I just had this in here because I was trying to debug the issue and was going to try and find a solution where it wasn't needed!